README.md

```diff
@@ -109,6 +111,5 @@ Please... Spread the word, contribute algorithms, submit performance improvement
 - [Scott Williams](https://github.com/tocsoft)
 - [Brian Popow](https://github.com/brianpopow)
 
-
-
+Thanks to [jetbrains](https://www.jetbrains.com) for supporting the ImageSharp team via the [Open Source Development Program](www.jetbrains.com/community/opensource/), so we can use the jetbrains tools for developing ImageSharp!
```

Comment:
Silk.NET have a good version of this at the bottom of their readme I think we should copy.

https://github.com/dotnet/Silk.NET/blob/90e4624fbbb1d068dd4a7203d411dfed301dde76/README.md

Reply:
Like this: 4a17bc53 ?